stonesoup/detector/tests/test_beamformers.py

```diff
@@ -0,0 +1,64 @@
+# -*- coding: utf-8 -*-
+import numpy as np
+from os import path
+from datetime import datetime
+
+from stonesoup.detector import beamformers
+from stonesoup.models.transition.linear import CombinedLinearGaussianTransitionModel, \
+                                               ConstantVelocity
+from stonesoup.models.measurement.linear import LinearGaussian
+from stonesoup.types.array import CovarianceMatrix
+from stonesoup.predictor.kalman import KalmanPredictor
+from stonesoup.updater.kalman import KalmanUpdater
+from stonesoup.types.state import GaussianState
+from stonesoup.types.hypothesis import SingleHypothesis
+from stonesoup.types.track import Track
+
+
+def fixed_target_beamformer_test():
```

Comment:
Looks like this didn't run. Test functions must start with `test_`
```suggestion
def test_fixed_target_beamformer():
```

Reply:
I've now corrected this (sorry, not familiar with PyTest).

I also noticed that I'd left in the hardcoded value of the signal frequency for testing, which I've now removed.